src/Text/Pandoc/Extensions.hs

```diff
@@ -601,6 +603,8 @@ getAllExtensions f = universalExtensions <> getAll f
   getAll "mediawiki"       = autoIdExtensions <>
     extensionsFromList
     [ Ext_smart ]
+  getAll "markua"          = extensionsFromList
+    [ Ext_citations]
```

Comment:
Is this a meaningful extension for Markua?  It doesn't have automatic citation support, does it?

Reply:
We changed it accordingly